stable_baselines/logger.py

```diff
@@ -124,8 +124,10 @@ def __init__(self, filename):
     def writekvs(self, kvs):
         for key, value in sorted(kvs.items()):
             if hasattr(value, 'dtype'):
-                value = value.tolist()
-                kvs[key] = float(value)
+                if value.shape == ():
+                    kvs[key] = float(value.tolist())
+                else:
+                    kvs[key] = value.tolist()
```

Comment:
missing comments

Reply:
Are you looking for a comment about this particular code block, or are you requesting I add a doc string to this method?

Reply:
small comment about those two cases, it is not trivial what value is

Reply:
yeah, i just made it a bit more robust as well